### PR TITLE
Update web-content-filtering.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line stating that "Only Microsoft Edge is supported  and Network Protection is not supported on AVD Windows 10 multisession session hosts." As only Edge works on AVD Windows 10 multisession session hosts, as they are listed as OS configuration: Member Server. This might not be obvious to IT admins who may run into the issues.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7673

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/web-content-filtering.md [hunk 1]
```diff
@@ -244,6 +244,8 @@ Use the time range filter at the top left of the page to select a time period. Y
 
 Only Microsoft Edge is supported if your device's OS configuration is Server (**cmd** \> **Systeminfo** \> **OS Configuration**). Network Protection is only supported in Inspect mode on Server devices, which is responsible for securing traffic across supported third-party browsers.
 
+Only Microsoft Edge is supported  and Network Protection is not supported on AVD Windows 10 multisession session hosts.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/web-content-filtering.md]:
```suggestion
Only Microsoft Edge is supported and Network Protection is not supported on Windows 10 Azure Virtual Desktop multi-session hosts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**